lib/plausible/ingestion/event.ex

```diff
@@ -76,6 +77,16 @@ defmodule Plausible.Ingestion.Event do
     {:ok, %{dropped: dropped, buffered: buffered}}
   end
 
+  defp check_engagements_flag(domain, %{event_name: "engagement"}) do
+    if FunWithFlags.enabled?(:engagements, for: %Plausible.Site{domain: domain}) do
```

Comment:
```suggestion
    if FunWithFlags.enabled?(:engagement_ingestion, for: %Plausible.Site{domain: domain}) do
```

This naming should make it clear what the flag is for.